CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Change the meaning of the `fee_amount` field inside `CoinSelectionResult`: from now on the `fee_amount` will represent only the fees asociated with the utxos in the `selected` field of `CoinSelectionResult`.
 - New `RpcBlockchain` implementation with various fixes.
 - Return balance in separate categories, namely `confirmed`, `trusted_pending`, `untrusted_pending` & `immature`.
+- Add capacity to create FeeRate from sats/kvbytes and sats/kwu.
+Rename `as_sat_vb` to `as_sat_per_vb`. Move all `FeeRate` test to `types.rs`.
```

Comment:
Tip: when you move some code without changing it, create a separated commit just for the move, like: https://github.com/bitcoin/bitcoin/commit/e93046c10b4d4e139cd7b41791ad1bfe925351e2

This makes reviewing the code and using `git blame` way easier!

You don't necessarily need to do this for this PR, just something to keep in mind for the future

Reply:
Thanks for the tip!